docs/source-pytorch/common/lightning_module.rst

```diff
@@ -84,39 +84,45 @@ Here are the only required methods.
 
 .. code-block:: python
 
-    import lightning.pytorch as pl
-    import torch.nn as nn
-    import torch.nn.functional as F
+    import pytorch_lightning as pl
```

Comment:
I think that the imports shall be consistent, so keep the import as it was

Reply:
updated in [648e829](https://github.com/Lightning-AI/lightning/pull/18451/commits/648e8295d62b3b6019388ed5119aa64be7d89143)